docs/get_started/centos_setup.md

```diff
@@ -0,0 +1,164 @@
+# Installing MXNet on CentOS
+MXNet currently supports Python, R, Julia, and Scala. For users on CentOS, MXNet provides [a docker installation guide](http://mxnet.io/get_started/docker_setup.html). If you don't have docker environment, please install MXNet step by step.
+
+Note: If you are not familiar with CentOS or building progress, we encourage you to [install or deploy MXNet environment on Ubuntu](http://mxnet.io/get_started/ubuntu_setup.html). 
+
+## Minimum Requirements
+Make sure you have the root permission, and `yum` is properly installed.
+
+To install MXNet on CentOS, you must have the following:
+
+1. gcc, g++ (4.8 or later)
+2. python2, python-numpy, python-pip, clang
+3. graphviz, jupyter (pip or yum install)
+4. OpenBLAS
+5. CUDA for GPU
+6. opencv (do not use yum to install opencv, some shared libs may not be installed) and cmake
+
+### Basic packages
```

Comment:
Maybe more intuitive title for this section would be - "Install Dependencies". And I think it should be at level 2.
And, Install OpenBLAS, Install CUDA for GPU and Install OpenCV can be sub-sections under this section.


## Install Dependencies
### Install OpenBLAS
### Install CUDA for GPU
### Install OpenCV